.github/workflows/dokka.yml

```diff
@@ -34,6 +34,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@4.1.4
         with:
```

Comment:
we don't want to overwrite main, just comment out as mentioned here, https://github.com/opensrp/fhircore/issues/1823#issuecomment-1315555032

```suggestion
#      - name: Deploy 🚀
#        uses: JamesIves/github-pages-deploy-action@4.1.4
#        with:
```